docs/docs/gatsby-image.md

```diff
@@ -0,0 +1,351 @@
+
+Once you've queried for a `fluid` image to retrieve its data, you can pass that data into the `Img` component:
+
+```jsx
+import { useStaticQuery } from "gatsby"
+import Img from "gatsby-image"
+
+export default () => {
+  const data = useStaticQuery(query`
+    file(relativePath: { eq: "images/default.jpg" }) {
+      childImageSharp {
+        # Specify a fluid image and fragment
+        # The default maxWidth is 800 pixels
+        // highlight-start
+        fluid {
+          ...GatsbyImageSharpFluid
+        }
+        // highlight-end
+      }
+    }
+  `)
+  return (
+    <div>
+      <h1>Hello gatsby-image</h1>
+      <Img
+        fluid={data.file.childImageSharp.fluid} {/* highlight-line */}
+        alt="Gatsby Docs are awesome"
+      />
+    </div>
+  )
+}
+```
+
+#### Fluid image query parameters
+
+In a query, you can specify options for fluid images.
+
+- `maxWidth` (int, default: 800)
+- `maxHeight`(int)
+- `quality` (int, default: 50)
+- `srcSetBreakpoints` (array of int, default: [])
+- `fit` (string, default: `[sharp.fit.cover][6]`)
+- `background` (string, default: `rgba(0,0,0,1)`)
+
+#### Returns
+
+- `base64` (string)
+- `src` (string)
+- `width` (int)
+- `height` (int)
+- `aspectRatio` (float)
+- `src` (string)
+- `srcSet` (string)
+
+This is where fragments like `GatsbyImageSharpFluid` come in handy, as they'll return all the above items in one line without having to type them all out:
+
+```graphql
+file(relativePath: { eq: "images/default.jpg" }) {
+  childImageSharp {
+    // highlight-start
+    fluid(maxWidth: 400) {
+      ...GatsbyImageSharpFluid
+      // highlight-end
+    }
+  }
+}
+```
+
+Read more in the [gatsby-plugin-sharp](/packages/gatsby-plugin-sharp/?=#fluid) README.
+
+### Resized images
+
+In addition to _fixed_ and _fluid_ images, the gatsby-image API allows you to call a `resize` method with `gatsby-plugin-sharp` to return a single image as opposed to multiple sizes. There are no default fragments available for the resize method.
+
+#### Parameters
+
+- `width` (int, default: 400)
+- `height` (int)
+- `quality` (int, default: 50)
+- `jpegProgressive` (bool, default: true)
+- `pngCompressionLevel` (int, default: 9)
+- `base64`(bool, default: false)
+
+#### Returns
+
+Resize returns an object with the following items:
+
+- `src` (string)
+- `width` (int)
+- `height` (int)
+- `aspectRatio` (float)
+
+```graphql
+allImageSharp {
+  edges {
+    node {
+        resize(width: 150, height: 150, grayscale: true) {
+          src
+        }
+    }
+  }
+}
+```
+
+### Shared query parameters
+
+In addition to `gatsby-plugin-sharp` settings in `gatsby-config.js`, there are additional query options that apply to both _fluid_ and _fixed_ images:
+
+- `grayscale` (bool, default: false)
+- `duotone` (bool|obj, default: false)
+- `toFormat` (string, default: ``)
```

Comment:
Also, this renders as an empty codeblock: 
![image](https://user-images.githubusercontent.com/21114044/59286361-d0244b80-8c2c-11e9-8252-056fa8335c34.png)

```suggestion
- `toFormat` (string, default: \`\`)
```

Maybe escape them so it would look like this: 
![image](https://user-images.githubusercontent.com/21114044/59286490-15487d80-8c2d-11e9-9d31-52358c082835.png)